scene/animation/animation_player.cpp

```diff
@@ -361,6 +361,15 @@ void AnimationPlayer::play_backwards(const StringName &p_name, double p_custom_b
 	play(p_name, p_custom_blend, -1, true);
 }
 
+void AnimationPlayer::play_with_capture(const StringName &p_name, double p_duration, Tween::TransitionType p_trans_type, Tween::EaseType p_ease_type, double p_custom_blend, float p_custom_scale, bool p_from_end) {
+	StringName name = p_name;
+	if (String(name) == "") {
```

Comment:
```suggestion
	if (name == StringName()) {
```
Cleaner, avoids conversion